### DISCUSSION TITLE
Tasks stuck in scheduled

### DISCUSSION THREAD

**Discussion by pauloventurab** [apache/airflow#33729] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

I'm a beginnier in Aiflow and I would like to know why my tasks that just print hello world in a single task is stuck on "scheduled" status. I'm running 50 in the same time, with parallelism=0 in the config and max_ative_runs = 50 and concurrency 50 with 9 workers and 1 scheduler.

Can you tell me please what could I do to improve it, it is too slow.

### What you think should happen instead

I would like to run all the 50 tasks in the same second.

### How to reproduce

Airflow 2.6.3
Kubernetes
Airflow cfg -> parallelism = 0
9 workers
1 scheduler
A dag with 1 taskapi to print hello world with max_active_runs = 50, concurrency = 50.

### Operating System

linux

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by hussein-awala** [comment]
Could you provide the log of your scheduler?

**Comment by nathadfield** [comment]
@pauloventurab With questions like this, I would suggest that you might better off joining the Airflow Slack where there's a whole channel dedicated to newbies.  Please refer to [this](https://airflow.apache.org/community/) for all the different ways you can engage with the community.
